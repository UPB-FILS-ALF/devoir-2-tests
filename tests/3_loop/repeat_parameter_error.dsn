& REPEAT parameter error

LOOP: 2
COLOR: fill 20 100 150
CIRCLE: 30 100 100 
REPEAT: 10
SET: %a 20
LOOP: 2
RECTANGLE: 10 20 100 20
ADD: %a 50
REPEAT
