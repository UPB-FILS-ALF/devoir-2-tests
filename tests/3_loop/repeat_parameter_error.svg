<?xml version="1.0" encoding="UTF-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1024" height="768">
	<!-- loop 2: 2 -->
	<!-- loop #0 -->
		<!-- fill 20, 100, 150 -->
		<circle cx="30" cy="100" r="100" stroke="rgb(0,0,0)"  stroke-width="1" fill="rgb(20,100,150)"/>
		<!-- a = 20 -->
		<!-- loop 2: 2 -->
		<!-- loop #0 -->
			<polygon points="10,20 100,20 100,20 10,20" stroke="rgb(0,0,0)"  stroke-width="1"  fill="rgb(20,100,150)"/>
			<!-- a = a + 50 : 70 -->
			<!-- repeat -->
		<!-- loop #1 -->
			<polygon points="10,20 100,20 100,20 10,20" stroke="rgb(0,0,0)"  stroke-width="1"  fill="rgb(20,100,150)"/>
			<!-- a = a + 50 : 120 -->
			<!-- repeat -->
</svg>
