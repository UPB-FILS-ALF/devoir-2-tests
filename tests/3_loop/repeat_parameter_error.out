ERROR LINE (6): REPEAT has 0 parameters, you wrote 1
ERROR LINE (3): LOOP with no REPEAT
